udata_hydra/analysis/csv.py

```diff
@@ -77,17 +77,14 @@
 minio_client = MinIOClient()
 
 
-async def notify_udata(check_id: int) -> None:
+async def notify_udata(check: Record | None, resource: Record | None) -> None:
     """Notify udata of the result of a parsing"""
-    # Get the check again to get its updated data
-    check: Record | None = await Check.get_by_id(check_id, with_deleted=True)
-    resource_id = check["resource_id"]
-    db = await context.pool()
-    record = await db.fetchrow("SELECT dataset_id FROM catalog WHERE resource_id = $1", resource_id)
-    if record:
+    if not check:
+        return
+    if resource:
```

Comment:
I'm confused, what would be the case without resource? And how would we send any info to udata if we don't have any resource?

Reply:
You're right, fixed